### PR TITLE
[5.4] Adding missing locales to Translation MessageSelector

### PR DESCRIPTION
**Pull request by jn-jairo**
Translations in the format below were returning the wrong plural with locales like `pt_BR`, because it does not find the locale in the getPluralIndex and always returns the default `0`.

```
'apples' => 'There is one apple|There are many apples',

echo trans_choice('messages.apples', 1);
// There is one apple

echo trans_choice('messages.apples', 0);
// There is one apple
```

### CODE REVIEW

**Comment by lucasmichot** [comment]
Should this not target `master` branch ?

File: src/Illuminate/Translation/MessageSelector.php [hunk 1]
```diff
@@ -111,116 +111,300 @@ public function getPluralIndex($locale, $number)
...
+            case 'tr_TR':
             case 'vi':
+            case 'vi_VN':
             case 'zh':
+            case 'zh_CN':
+            case 'zh_HK':
+            case 'zh_SG':
+            case 'zh_TW':
                 return 0;
                 break;
```
Comment (lucasmichot) [inline src/Illuminate/Translation/MessageSelector.php]: useless `break`

File: src/Illuminate/Translation/MessageSelector.php [hunk 2]
```diff
@@ -111,116 +111,300 @@ public function getPluralIndex($locale, $number)
...
+            case 'hi_IN':
             case 'hy':
+            case 'hy_AM':
             case 'ln':
+            case 'ln_CD':
             case 'mg':
+            case 'mg_MG':
             case 'nso':
+            case 'nso_ZA':
             case 'xbr':
```
Comment (lucasmichot) [inline src/Illuminate/Translation/MessageSelector.php]: `wbr` should be after `wa_BE`

**Comment by jn-jairo** [comment]
@lucasmichot Thanks for your review.

I don't know if it should target the `master` branch.

I'm using the version `5.4` and the `trans_choice` is not working for my locale `pt_BR`, so i fixed it, and target the `5.4` branch because I need this fix in the version `5.4`.

If I need to target the `master` branch, I can do it, I just need to be sure that the `master` branch is the correct to make this change applied to `5.4` too.

**Comment by taylorotwell** [comment]
5.4 is fine. It's basically a bug.

**Pull request merged**